pico-statsd/src/test/scala/org/pico/statsd/syntax/event/PerformanceMain.scala

```diff
@@ -0,0 +1,32 @@
+package org.pico.statsd.syntax.event
+
+import com.timgroup.statsd.StatsDClient
+import org.pico.disposal.{Auto, Disposable}
+import org.pico.event.Bus
+import org.pico.statsd.PicoStatsDClient
+
+import scala.concurrent.duration.Deadline
+
+object PerformanceMain {
```

Comment:
This is used for performance testing.  Can be run in a profiler on local development environment.